### ISSUE TITLE
How to give gnuroot debian root access?

### ISSUE THREAD

**Issue by Root-voidX** [corbinlc/GNURootDebian#227]
This might be a silly question, but how can i give the proot root access? Has anyone done it? I am wondering if giving it root access will allow me to get more use and maybe circumvent selinux or actually run certain applications (gmsh and scilab) without it throwing me a segmentation fault (which I assume is due to permissions)?

**Comment by ksaves2** [comment]
Don't think it's possible.  I've been trying to get access to an attached B/T serial device (not a mouse or a keyboard as they are automatically there if attached on the Android side) and nothing has been posted.
One can't access /dev to see what's in there or alter any permissions to access a "paired" device.
Really handicaps me for what I want to do.  I like the concept of running Jessie on top of the Android device.  It means I don't have to ditch the Android OS or coapt it to run the Nexus 7 as a pure Linuxbox.
Kurt

**Comment by Root-voidX** [comment]
@ksaves2
I read under the proot github that you can indeed give it root access. Idk how to do it via gnuroot Debian as either the commands do nothing or terminal freezes. I also would of loves to attach usb sensors to read into octave/scilab or run some simulations. 

Either hopefully the next build of gnuroot debian helps keep selinux out of the proot, or I guess we have to try our own proot or use Linux deploy. 

What are you trying to do with gnuroot exactly? Kind of curious

**Comment by ksaves2** [comment]
Hi, Thanks for responding,
I'd like to bond to an HC-05 or HC-06 B/T serial module and pipe the incoming strings to the APRS program called Xastir.  In a native linux box a bonded B/T device is known as /dev/rfcomm"X".
Sometimes I have to do a sudo chmod on it but I can usually get the device permission and 
assign the port in Xastir for use.  I can use my devices internal GPS via gpsd using the Android GPS share program.  If I want to go off Wi-Fi and use the cellphone -IS (internet server) service I can get it to work but I can't get find where the outboard B/T device is at or if the permission is just permanently 
inaccessible.
There is also a bluetooth TNC called a Mobilnkd: http://www.mobilinkd.com/tnc2/ which does have Android program access but can be used under native a Linux system.

I am a bit puzzled in that a B/T keyboard and or mouse paired under Android is available to be used 
under GRD.  Any way to find or delineate other devices?  That's the kicker here.

Oh, one other thing with running Xastir with XSDL is that I can manipulate Xastir with my finger and touchscreen only.  Can't do that on any of my laptops and must use a mouse of the touchpad.  Kurt KC9LDH   https://aprs.fi/#!lat=40.53970&lng=-90.05850

**Comment by AdamMatthewLTM** [comment]
Hi all,

I am currently assisting the developers in sorting relevant issues for GNURoot Debian on GitHub. If the issue discussed is still relevant, could you please reply to this thread and state the issue needs to be resolved and I will attempt to sort it for you ASAP. If the issue is no longer relevant, please reply to this thread and state it has been resolved, and I will suggest this thread be closed.

Furthermore, if a particular issue is considered to be high priority, please state that you are requesting it to be considered as high priority and I will mark it as such.

Thank you all so much for your time and understanding,
-Adam

**Comment by ksaves2** [comment]
Copy and posted in another thread here too:
Ok Adam,

Let me point it out this way. GRD very nicely has access to a Bluetooth keyboard and or Bluetooth mouse that has been previously paired on the Android side. Just pair them " over there" and they're
ready for use in GRD/LXDE. Now, there are B/T serial devices that can be paired to deliver a simple datastream to the kernel. There is no facility to access these devices in GRD though they may be paired on the Android side. If a "pure" linux laptop were used they reside at /dev/rfcomm"X"
(/dev/rfcomm0 for example) and are fully usable in a native linux distro.
Now lacking this ability, I went to the android app "Linux Deploy" and was able to get the base
Kali Linux distro on my Nexus 7 2013. I'm not interested in the "intrusion stuff" and it's not in there
thankfully. I thought there would be perhaps easier access to /dev and though there appears to be access to the directory, still no ability to pair/access other Bluetooth devices. I posted this in the
LD Kali group and a couple of folks posted they had to write some rough code themselves to acheive this but it's really hard to use.
Basically, if you could get GRD to access other B/T devices on demand it would be the first "Linux on Android" that could do that and you would be in a better position to charge a subscription fee for the distro. I'd pay for it.
Oh, another thing, fix it to work in an 8.1 Oreo kernel please. Linux Deploy won't work there either so
that would be a jump over them. Kurt KC9LDH
